knackpy/app.py

```diff
@@ -205,7 +205,7 @@ def get(
                     API. Defaults to False.
                 record_limit (int): the maximum number of records to retrieve. If
                     `None`, will return all records.
-                filters (dict or list, optional): A dict or of Knack API filiters.
+                filters (dict or list, optional): A dict or of Knack API filters.
```

Comment:
I actually have a second typo here (lol 🙈): the word "list" is missing.

> filters (dict or list, optional): A dict or _list_ of Knack API filters.

Reply:
My reliance on my spell checker in VS Code has never been more apparent! Without a squiggly line, I didn't even see that!